pylint_odoo/test/main.py

```diff
@@ -41,8 +41,8 @@
     'method-inverse': 1,
     'method-required-super': 8,
     'method-search': 1,
-    'missing-import-error': 4,
-    'missing-manifest-dependency': 2,
+    'missing-import-error': 7,
```

Comment:
If we exclude test directory, why the number of errors raise?

Reply:
Because we're also adding 6 new cases: 3 in the tests directory and 3 more in a model file. Then, there should be only 3 more cases.